parameter/SystemClass.cpp

```diff
@@ -208,14 +168,23 @@ bool CSystemClass::loadPlugins(list<string>& lstrPluginFiles, core::Results& err
         string strPluginFileName = *it;
 
         // Get plugin symbol
-        string strPluginSymbol = getPluginSymbol(strPluginFileName);
+#ifndef PARAMETER_FRAMEWORK_PLUGIN_ENTRYPOINT_V1
+#   error Missing PARAMETER_FRAMEWORK_PLUGIN_ENTRYPOINT_V1 macro definition
+#endif
+
+#define QUOTE(X) #X
+#define MACRO_TO_STR(X) QUOTE(X)
+        string strPluginSymbol = MACRO_TO_STR(PARAMETER_FRAMEWORK_PLUGIN_ENTRYPOINT_V1);
+#undef MACRO_TO_STR
+#undef QUOTE
```

Comment:
Should this not be done in the CSubsystemLibrary ? Ie have a static const char * with the name. In the same way you have a type alias SubsystemBuilderFunction